### PR TITLE
V0 blogpost

### PR DESCRIPTION
**Pull request by orianac**
Starting the PR for these two notebooks and the changes to the README. @tcchiao and @jhamman your 👀 would be great! I'll add the catalog to this as well. On the notebooks, I might only highlight the `blogpost_sample_notebook.ipynb` - the `raster_packaging.ipynb` is a little behind the scenes.

### CODE REVIEW

File: carbonplan_trace/v0/data/catalog.yaml [hunk 1]
```diff
@@ -47,8 +47,8 @@ sources:
     args:
       urlpath: 'http://gfw2-data.s3.amazonaws.com/climate/WHRC_biomass/WHRC_V4/Processed/{{ lat }}_{{lon}}_t_aboveground_biomass_ha_2000.tif'
       chunks: {'y': 4000, 'x': 4000}
-
-  gwf_emissions:
+      
+gwf_emissions:
```
Comment (tcchiao) [inline carbonplan_trace/v0/data/catalog.yaml:51]: not a big deal but typo here gwf -> gfw

File: carbonplan_trace/v0/data/catalog.yaml [hunk 2]
```diff
@@ -71,3 +71,81 @@ sources:
     args:
       urlpath: 'https://data-api.globalforestwatch.org/dataset/gfw_forest_carbon_gross_emissions/v20210331/download/geotiff?grid=10/40000&tile_id={{ lat }}_{{lon}}&pixel_meaning={{ unit }}'
       chunks: {'y': 4000, 'x': 4000}
+
+emissions_30m_partitioned:
+    description: Global 30m emissions from CarbonPlan partitioned into fire and non-fire related components
+    metadata:
+      url: ''
```
Comment (tcchiao) [inline carbonplan_trace/v0/data/catalog.yaml:78]: perhaps link to the blog post or methods doc?

File: carbonplan_trace/v0/data/catalog.yaml [hunk 3]
```diff
@@ -71,3 +71,81 @@ sources:
...
+      tags:
+        - emissions_partitioned
+    parameters:
+      lat:
+        description: latitude-tile
```
Comment (tcchiao) [inline carbonplan_trace/v0/data/catalog.yaml:83]: of the northwest corner of 10x10 degree tile

File: carbonplan_trace/v0/data/catalog.yaml [hunk 4]
```diff
@@ -71,3 +71,81 @@ sources:
...
+        description: emission units
+        type: str
+        default: Mg_CO2e_ha
+    driver: rasterio
+    args:
+      urlpath: 'https://carbonplan-climatetrace.s3.us-west-2.amazonaws.com/v0.4/tiles/30m/{{ lat }}_{{ lon }}.zarr'
+      chunks: {'y': 4000, 'x': 4000}
+      
+emissions_3km:
+    description: Global 3 km emissions from CarbonPlan partitioned into fire and non-fire related components
```
Comment (tcchiao) [inline carbonplan_trace/v0/data/catalog.yaml:124]: this description and the `emissions_3km_partitioned` description below are swapped (or name is swapped)

File: notebooks/blogpost_sample_notebook.ipynb [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+   "id": "75d9feed-9bc7-4fbe-92d0-a7150011a946",
+   "metadata": {},
+   "source": [
+    "<img width=50 src=\"https://carbonplan-assets.s3.amazonaws.com/monogram/dark-small.png\" style=\"margin-left:0px;margin-top:20px\"/>\n",
+    "\n",
+    "# Sample notebook to explore biomass dataset\n",
+    "\n",
+    "Authors: Oriana Chegwidden and Cindy Chiao\n",
+    "\n",
+    "To run this notebook locally and produce the loveliest-possible figures, you'll\n",
```
Comment (tcchiao) [inline notebooks/blogpost_sample_notebook.ipynb:14]: :D

File: notebooks/blogpost_sample_notebook.ipynb [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "80d9a4fe-6878-42b4-800d-a12b4d9c86bb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path_template = (\n",
+    "    \"s3://carbonplan-climatetrace/{version}/tiles/30m/{tile}_tot.zarr/\"\n",
```
Comment (tcchiao) [inline notebooks/blogpost_sample_notebook.ipynb:160]: should we update this to use the catalog instead?

**Comment by tcchiao** [comment]
I think it'd also be helpful to link to the blog post from this page since we put this link on the climate trace website

File: carbonplan_trace/v0/data/catalog.yaml (hunk 1 above)
Comment (orianac) [reply carbonplan_trace/v0/data/catalog.yaml:51]: nice catch

File: carbonplan_trace/v0/data/catalog.yaml (hunk 2 above)
Comment (orianac) [reply carbonplan_trace/v0/data/catalog.yaml:78]: perfect

File: carbonplan_trace/v0/data/catalog.yaml (hunk 3 above)
Comment (orianac) [reply carbonplan_trace/v0/data/catalog.yaml:83]: super

File: carbonplan_trace/v0/data/catalog.yaml (hunk 4 above)
Comment (orianac) [reply carbonplan_trace/v0/data/catalog.yaml:124]: eagle eyes!

File: notebooks/blogpost_sample_notebook.ipynb (hunk 5 above)
Comment (orianac) [reply notebooks/blogpost_sample_notebook.ipynb:14]: 💃

File: notebooks/blogpost_sample_notebook.ipynb (hunk 6 above)
Comment (orianac) [reply notebooks/blogpost_sample_notebook.ipynb]: done! so slick.

**Pull request merged**